definitions/checks/disk/performance.rb

```diff
@@ -1,9 +1,16 @@
 module Checks
   module Disk
     class Performance < ForemanMaintain::Check
+      DEFAULT_DIRS = %i[pulp mongo foreman_database].inject({}) do |dirs, f|
+        if feature(f) && File.directory?(feature(f).data_dir)
```

Comment:
I found out this actually breaks feature loading. While loading definitions we require this check which tries to evaluate the features. There are none loaded yet at the moment and it sets the feature singleton to empty hash. There is no way to update it later. Therefore we need to evaluate this later.

This worked for me:
```ruby
      def self.default_dirs
        @default_dirs ||= %i[pulp mongo foreman_database].inject({}) do |dirs, f|
          if feature(f) && File.directory?(feature(f).data_dir)
            dirs[feature(f).label_dashed] = feature(f).data_dir
          end
          dirs
        end
      end

      def description
        "Check recommended disk speed for #{self.default_dirs.keys.join(', ')} services."
      end
```
and I had to remove the description from metadata and fix the other references to the DEFAULT_DIRS

Reply:
@mbacovsky thank you for suggestions. The self.default_dirs call does not work, did you miss 'class' keyword inbetween ? I have pushed the changes with 'self.class.default_dirs' which is working for me. Unfortunately there are few tests errors, I should look into that asap.

Reply:
@upadhyeammit the typo is possible I was moving the code around. I guess it could work well enough if it is even instance a method (no self.) and not a class method. So feel free to remove it if it feels cleaner.